### DISCUSSION TITLE
请教下ExH增强器的cookie格式

### DISCUSSION THREAD

**Discussion by Tandio** [anobaka/Bakabase#236] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/49389044/187203109-38879719-6b72-4794-9df2-9eef2b8d8f98.png)
顺便问一下，这个exh增强器的搜索是全文件名搜索么

**Comment by anobaka** [comment, marked as answer]
向exhentai发起请求时使用的完整cookie header的value，可以通过浏览器的【开发者工具】-【网络】-访问exhentai-【在网络中点选对应的请求】-【右侧请求详情header里的一串cookie字符串】，稍后我会补充文档。

> 顺便问一下，这个exh增强器的搜索是全文件名搜索么

优先级`资源名称`>`文件名`，默认情况下如果不开启任何增强器，是不会填充`资源名称`的，可以先用`insideworld增强器`解析`资源名称`（部分情况下可能会解析异常）

**Reply by anobaka** [reply]
文档已更新
http://inside-world.anobaka.com/doc#category-enhancer-exhentai
